(GH-10088) Add example for normalizing `Read-Host` input (#10089)
MicrosoftDocs/PowerShell-Docs: reference/7.3/Microsoft.PowerShell.Utility/Read-Host.md

the entries. This could lead to an error later in the script where the code expects an exact
name.

The example shows how you can convert an input string into an array of entries without any
extra spaces.

```powershell
$prompt = @(
    'List the cities you want weather information for.'
    'When specifying multiple cities, separate them with a semi-colon, like:'
    "'New York; Osan; Koforidua'"
) -join ' '

$cities = Read-Host $prompt

"Input cities string: `n`t'$cities'"

$splitCities = $cities -split ';'

"Split cities array:"
$splitCities | ForEach-Object -Process { "`t'$_'" }

$normalizedCities = $splitCities  | ForEach-Object -Process { $_.Trim() }

"Normalized split cities array:"
$normalizedCities | ForEach-Object -Process { "`t'$_'" }
```

```output
Input cities string:
        '    New York;  Osan   ;Koforidua   '
Split cities array:
        '    New York'
        '  Osan   '
        'Koforidua   '
Normalized split cities array:
        'New York'
        'Osan'
        'Koforidua'
```

The example uses the `-split` operator to convert the input string into an array of strings. Each

> string in the array includes the name of a different city. However, the split strings still include
> extra spaces. The example uses the **Trim** method on each string in the array to remove the
> spaces.

Reviewer: ```suggestion
string in the array includes the name of a different city. However, the split strings include extra
spaces. The `Trim()` method removes the leading and trailing spaces from each string.
```
